realtime_updater.py

```
@@ -0,0 +1,119 @@
+                    "team": row.get("team"),
+                    "position": row.get("position"),
+                    "rush_yds": row.get("rushing_yards", 0),
+                    "rush_att": row.get("rushing_attempts", 0),
+                    "rec_yds": row.get("receiving_yards", 0),
+                    "rec": row.get("receptions", 0),
+                    "tgt": row.get("targets", 0),
+                },
+            )
+        conn.commit()
+        logger.info("Inserted %d live stat rows", len(stats))
+    except Exception as exc:
```

Pylint (3.3.7)

[warning] Catching too general exception Exception

(W0718)